src/unitxt/inference.py

```diff
@@ -250,6 +253,47 @@ def _infer(self, dataset):
             )
         ]
 
+    def _infer_log_probs(self, dataset):
+        from genai.schema import TextGenerationParameters
+
+        logprobs_return_options = {
+            "generated_tokens": True,
+            "input_text": False,
+            "input_tokens": False,
+            "token_logprobs": True,
```

Comment:
@elronbandel  we should probably have this argument configurable in the abstract class somehow